.storybook/preview.tsx

```diff
@@ -41,39 +41,48 @@ const fontLoader = async () => ({
 const colorThemeConfig = {
   light: {
     'brand-colors': {
-      'brand-color': '#0073ea',
-      'brand-hover-color': '#0060b9',
+      'brand-color': '#7822FF',
+      'brand-hover-color': '#10181B',
```

Comment:
I think for hover color there is a mistake by accident (in Craig's Figma) -- black should not replace purple, it is against standard and best UIUX practices.
`#10181B` should be replaced with `#5605c0`
![image](https://github.com/Ikigai-Labs-IO/monday-ui-react-core/assets/5389749/42df3502-fc11-45af-90fa-e9010e65332c)